packages/components/src/mobile/html-text-input/index.native.js

```diff
@@ -117,7 +116,8 @@ export default compose( [
 				editPost( { content } );
```

Comment:
This is not related to your changes, but I'm not sure that this `editPost( { content } )` call is actually doing anything since it doesn't seem like . Just curious if you happen to know of something I'm missing there.

Reply:
Well it will still update the `content` value of the post in the store which is used to undo/redo, however it will not be used by the main selector https://github.com/WordPress/gutenberg/blob/master/packages/editor/src/store/selectors.js#L878 as it checks if `record.blocks` is defined first